agent/tool-scripts/postprocess/pidstat-postprocess

```diff
@@ -82,34 +81,6 @@ if (scalar @ARGV > 2) {
 		}
 	}
 	close FH;
```

Comment:
This whole section got deleted, presumably because it should go into pidstat-stop-process, but there is no pidstat-stop-postrpocess in the commit.

Reply:
Actually, I could not find where `$vcpus` was being used in the original code.  So I just removed it entirely.